pkgs/applications/search/catfish/default.nix

```diff
@@ -45,6 +45,9 @@ pythonPackages.buildPythonApplication rec {
     done
   '';
 
+  # Disable tests, they don't exist and break the builds (according to debian)
+  doCheck = false;
```

Comment:
Better check yourself. Are there tests or are there no tests?

Reply:
Testing gives the following errors:
```
running install tests
ERROR: Python module helpers not found
ERROR: Python module Window not found
ERROR: Python module catfishconfig not found
ERROR: Python module catfishconfig not found
ERROR: Python module Builder not found
ERROR: Python module helpers not found
ERROR: Python module catfishconfig not found
ERROR: Python module zeitgeist.client not found
ERROR: Python module zeitgeist.datamodel not found
ERROR: Python module zeitgeist not found
ERROR: Python module helpers not found
```

Those modules are installed at `$out/lib/python2.7/site-packages/catfish_lib/`.

Reply:
Looking at the tarball, I don't see any test. Do you know what command needs to be run for the tests? If not, I think you can honestly say there are no tests. Please leave a comment why you disable them in this case.

Reply:
I also do not see any test in the tarball.